src/components/Calendar/doubleCalendar/yearSelect/index.js

```diff
@@ -49,24 +65,39 @@ export default function YearSelect(props) {
                 ref={selectRef}
                 value={currentYear}
                 editMode={isEditMode}
+                conClick={onClick}
                 onChange={handleYearChange}
                 onFocus={handleSelectFocus}
                 onBlur={handleSelectBlur}
+                tabIndex={tabIndex}
+                onKeyDown={onKeyDown}
```

Comment:
same ^^ If we finally don't add onKeyDown to select then this is likely not used, if this is true the remove and clean this